.github/workflows/pull-request.yml

```diff
@@ -0,0 +1,58 @@
+name: DualGovernance PR tests
+
+permissions:
+  contents: read
+
+on:
+  pull_request:
+    branches: ['main', 'develop']
```

Comment:
In some cases, long-lived branches targeted by other PRs may appear, and restricting CI to only two branches can complicate the repository workflow. Consider running these tests on pull requests from any branch.